rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -119,6 +119,16 @@ def ensure_global(name, default):
     # TODO(aprotyas): Migrate files under the `include` tag in the project's Doxygen
     # configuration into the Sphinx project tree used to run the Sphinx builder in.
     extensions.append('myst_parser')
+
+# if no `primary_domain` option provided, defaults to 'cpp'
+if rosdoc2_settings.get('primary_domain', 'cpp'):
```

Comment:
This condition is a bit weird? Basically it would only be false if the user explicitly did `rosdoc2_settings = {'primary_domain': None}` or something like that.

Reply:
I concur, I've defaulted to `None` rather than `cpp` for this condition in ac4a5ea595ffd00a64d99c7a7eed4dc62bc773a9.